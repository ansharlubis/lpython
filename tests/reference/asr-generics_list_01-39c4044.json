{
    "basename": "asr-generics_list_01-39c4044",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/generics_list_01.py",
    "infile_hash": "cd02f7ec58b9eb8ecabd420541968b7a4065affbbaffb02c91e999fc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-generics_list_01-39c4044.stderr",
    "stderr_hash": "322e871cca896d9c22b541f49a5061305aa099eb4d76035625006a46",
    "returncode": 2
}