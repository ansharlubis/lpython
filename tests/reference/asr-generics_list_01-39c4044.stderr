semantic error: Binary operations are not allowed between non-number variables
 --> tests/../integration_tests/generics_list_01.py:6:15
  |
6 |         acc = acc + lst[i]
  |               ^^^^^^^^^^^^ 
